api/src/main/java/org/eclipse/microprofile/faulttolerance/Retry.java

```diff
@@ -97,14 +97,16 @@
 
     /**
      *
-     * @return Specify the failure to retry on
+     * @return Specify the failure to retry on. Only a type of {@link java.long.Error} or {@link java.lang.Exception} can be specified.
```

Comment:
I think it should be something like: Only a type assignable from `java.long.Error` or `java.lang.Exception` can be used.